configure.ac

```diff
@@ -561,6 +561,39 @@ case "$enable_unwind_ptrace" in
       ;;
 esac
 
+AC_ARG_ENABLE([libiberty],
```

Comment:
Autoconf naming convention typically avoids "--enable-LIBRARY" for library support. You have two ways to fix this:
1. Use "--enable-FEATURE" naming such as "--enable-demangle". I personally recommend this way as people can add demangling support for other OSes.
2. Change the option name to "--with-libiberty". See [`AC_ARG_WITH` in the Autoconf manual to know what I mean.

Reply:
I choose the first one. I let you solve the conversation if the change is correct for you

Reply:
(Please help me close this conversation. I cannot close it myself and there's another conversation below this one. This this conversation can be closed as resolved/ outdated.)